packages/ui/src/views/layouts/PullRequestLayout.tsx

```diff
@@ -66,10 +66,8 @@ export const PullRequestLayout: FC<PullRequestLayoutProps> = ({
         <Tabs.Root variant="tabnav" className="mb-7" defaultValue={PullRequestTabsKeys.CONVERSATION}>
           <Tabs.List
             className={cn(
-              'before:w-[calc(100vw-var(--sidebar-width))] before:min-w-[calc(100%+3rem)] before:left-1/2 before:-translate-x-1/2',
-              {
-                'before:w-[calc(100vw-var(--sidebar-width)-6px*2)]': isInset
-              }
+              'before:w-[calc(100vw-var(--sidebar-width))] before:min-w-[calc(100%+3rem)] before:left-1/2 before:-translate-x-1/2 before:bg-background-9',
```

Comment:
the `calc(100%+3rem)` for min-width looks suspiciously specific to a screen size. is this tested for different resolution screens?

Reply:
Absolutely
This is the paddings compensation, so the bottom border of the tabs can go through paddings
<img width="1016" alt="image" src="https://github.com/user-attachments/assets/979e11c0-39e5-4965-955a-168eb6934a8f" />

This isn't a new code, it's already works this way